cdk/src/open-data-platform/lambda/data-import-stack.ts

```diff
@@ -55,8 +61,11 @@ export class DataImportStack extends Construct {
 
     if (writeDemographicDataFunction.role?.roleArn != undefined) {
       let role = iam.Role.fromRoleArn(
-        scope, id + '-role', writeDemographicDataFunction.role.roleArn);
+        scope,
+        id + '-role',
```

Comment:
nit: I find string templates like `` `${id}-role` `` to be more readable than using the `+` operator, which might do some weird type casting.